gsuite_activityevent_rules/gsuite_brute_force_login.yml

```diff
@@ -0,0 +1,58 @@
+AnalysisType: rule
+Filename: gsuite_brute_force_login.py
+RuleID: GSuite.BruteForceLogin
+DisplayName: --DEPRECATED-- GSuite Brute Force Login
```

Comment:
Not sure why this is deprecated, will look into it

Reply:
Likely because its covered by a [standard rule](https://github.com/panther-labs/panther-analysis/blob/master/standard_rules/brute_force_by_ip.yml#L10)